fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadTaskInfo.java

```diff
@@ -66,6 +67,10 @@
 
     protected long timeoutMs = -1;
 
+    // this status will be set when corresponding transaction's status is changed.
+    // so that user or other logic can know the status of the corresponding txn.
+    protected TransactionStatus txnStatus = TransactionStatus.UNKNOWN;
```

Comment:
Is it useful?

Reply:
Yes, if it is set to COMMITTED, the timeout checker will not handle this task

Reply:
Yes, if it is set to COMMITTED, the timeout checker will not handle this task